plugin/kubernetes/controller_test.go

```diff
@@ -79,6 +79,9 @@ func generateEndpoints(cidr string, client kubernetes.Interface) {
 		}
 		ep.ObjectMeta.Name = "svc" + strconv.Itoa(count)
 		_, err = client.CoreV1().Endpoints("testns").Create(ep)
+		if err != nil {
```

Comment:
this should return the err; but this being test code I don't think we care **that** much

Reply:
idem for the other test functions